### PR TITLE
added training data

### LINKED ISSUE #43: Need to generate training data in the json format
Recently I tested NN with big number of neurons (#42). It worked quite well, but the test case contained only three letters.

I think we can consider it as a starting point. There is no deal breaker to test NN with real data.

But! NN will contain 16*16 input neurons, about 150 hidden neurons and something more output neurons. It's quite many! And we can't test all the time when we run "tox" because it'd take us hours to wait for test's ending.

So I suppose we're ready for smooth transition from local working to interaction with the server. We'll teach NN on the server.

Of course, there are a lot of work to do it. And I should make a little more preparations of NN's methods (but that's another subject).

**TL;DR:**
Anyway first important thing is training dataset (TD). Look at small prototype of TD: [nn_aye.json](https://github.com/ssu-451/project/blob/master/ocr/tests/nn_aye.json)

That's a small TD for only three letters. I made it manually. Now we need the same TD with all letters.

I suggest you use only Latin letters. Don't give a fuck about Russian letters. There's not a point. It'll just slow down NN's work.

So ___the task___: you should generate json file in the [following format](https://github.com/ssu-451/project/blob/master/ocr/tests/nn_aye.json) for the following symbols:
```
ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789 .,"?'!-
```

**Assignees**: @Andy-ch, @KolesnikovaTamara, @WalkingMike 
**Labels**: feature request
**Milestone**: 2018.03.20

### PR DESCRIPTION
**Pull request by WalkingMike**
added training data for symbols ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789 .,"?'!-
close #43

### CODE REVIEW

**Comment by WalkingMike** [comment]
It was done exactly like that - human-readable. Feel at ease to tell me what do i need to change if it is so inconvenient for processing by machine .

File: ocr/tests/symbols_data.json [hunk 1]
```diff
@@ -0,0 +1,1335 @@
...
+                      0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0,
+                      0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0,
+                      0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0,
+                      0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        {
+            "letter": "Lbig",
```
Comment (ferc255) [inline ocr/tests/symbols_data.json]: L

File: ocr/tests/symbols_data.json [hunk 2]
```diff
@@ -0,0 +1,1335 @@
...
+                      0, 0, 0, 0, 0, 1, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        {
+            "letter": "exclamation",
```
Comment (ferc255) [inline ocr/tests/symbols_data.json]: !

File: ocr/tests/symbols_data.json [hunk 3]
```diff
@@ -0,0 +1,1335 @@
...
+                      0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0,
+                      0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        {
+            "letter": "hyphen",
```
Comment (ferc255) [inline ocr/tests/symbols_data.json]: "-"

File: ocr/tests/symbols_data.json [hunk 4]
```diff
@@ -0,0 +1,1335 @@
...
+                      0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 1, 1, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        {
+            "letter": "single_quote",
```
Comment (ferc255) [inline ocr/tests/symbols_data.json]: "'"

File: ocr/tests/symbols_data.json [hunk 5]
```diff
@@ -0,0 +1,1335 @@
...
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        {
+            "letter": "question",
```
Comment (ferc255) [inline ocr/tests/symbols_data.json]: ?

**Comment by WalkingMike** [comment]
Such order generated automatically. So do you want human-readable or not? If it is processing by machine - what does the order mean?
And ' with " need to be like \" and \' , am i right, @ferc255 ?

**Comment by ferc255** [comment]
@WalkingMike, essentially, I won't use "letter" field at all. It's just for the beauty and readability.

**Comment by Andy-ch** [comment]
@ferc255 I don't like that. It won't be obvious how to read those data if you remove the letters at all. Again, I suggest dictionary where letters would be keys

**Comment by WalkingMike** [comment]
In my opinion dictionary is better-looking. So do we need to change format for simplicity(dictionary akin "A": dataA, "B":dataB)?

**Comment by Andy-ch** [comment]
I think so, but it seems there is no total agreement

File: ocr/tests/symbols_data.json [hunk 6]
```diff
@@ -0,0 +1,1336 @@
...
+                      0, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 0,
+                      0, 0, 0, 1, 0, 0, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0,
+                      0, 0, 1, 1, 0, 0, 0, 1, 1, 0, 0, 0, 1, 1, 0, 0,
+                      0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0,
+                      0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0,
+                      0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+N
```
Comment (ferc255) [inline ocr/tests/symbols_data.json]: ???

**Comment by WalkingMike** [comment]
There was nothing. It was not the mistake you were looking for.
